pom.xml

```diff
@@ -255,6 +258,10 @@
                   <group>
                     <title>Azure Key Vault</title>
                     <packages>com.azure.security.keyvault*</packages>
+                  </group>
+				  <group>
```

Comment:
nit: indention

Reply:
I noticed the indentation changes - my editor was not correctly set up. Let me know if it is causing problems somewhere and I can send a separate PR or else will club this with another PR I am about to send!

Reply:
I don't think it hurts anything I just noticed it while scanning the PR.